gormigrate.go

```diff
@@ -178,7 +194,7 @@ func (g *Gormigrate) createMigrationTableIfNotExists() error {
 		return nil
 	}
 
-	sql := fmt.Sprintf("CREATE TABLE %s (%s VARCHAR(255) PRIMARY KEY)", g.options.TableName, g.options.IDColumnName)
+	sql := fmt.Sprintf("CREATE TABLE %s (id INTEGER PRIMARY KEY AUTOINCREMENT, %s VARCHAR(255))", g.options.TableName, g.options.IDColumnName)
```

Comment:
The reason I didn't had this in the first place, is because it's not compatible with different databases. E.g. it could be `AUTOINCREMENT`, `ANTO_INCREMENT`, type `SERIAL` for PostgreSQL, etc.

It's also backwards incompatible, so let's keep as is.